src/identifier/mod.rs

```diff
@@ -200,32 +202,6 @@ impl Identifier {
     }
 }
 
-// Identifier implementation for wasm
-#[cfg(target_arch = "wasm32")]
-impl Identifier {
-    // There is no file system on the web, so we are not reading strings from file.
-    // let the user perform the I/O and read the file, then pass the content of it.
-    pub fn identify(&self, text: &[String]) -> Vec<Match> {
```

Comment:
This was required due to the reasons mentioned in comment.

Reply:
fixed